### DISCUSSION TITLE
Official Gatsby Plugin for AirTable

### DISCUSSION THREAD

**Discussion by ZeldOcarina** [gatsbyjs/gatsby#37470] [Ideas / Feature Requests] [answered]
Hi wonderful team,

Unfortunately the [gatsby-source-airtable](https://github.com/jbolda/gatsby-source-airtable) community plugin is completely unmantained with the latest commit having been well over seven months ago.

There is some community alternatives but I think AirTable is in such a wide use now that it would deserve to enter the core plugins.

The current plugin is well written, it might be cloned and updated and it might not bee too much of a pain to do.

Since AirTable is thriving these days and it works amazingly well as a CRM for websites, I think it could be cool to give it some consideration:

https://www.mksguide.com/airtable-user-and-company-stats/

What do you think?

**Comment by LekoArts** [comment, marked as answer]
Hi!

We won't be adding any other plugins to this monorepo here but there's a wonderful project: https://github.com/gatsby-uc/plugins
Once you reach the maintainer of the airtable plugin, they could transfer it to that organization and you (and anyone else) can maintain it :)
